integration/mdc-migration/golden/src/app/components/paginator/paginator.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+
+import {PaginatorComponent} from './paginator.component';
+
+describe('PaginatorComponent', () => {
+  let component: PaginatorComponent;
+  let fixture: ComponentFixture<PaginatorComponent>;
+
+  beforeEach(async () => {
+    await TestBed.configureTestingModule({
+      declarations: [PaginatorComponent],
```

Comment:
Do we need the paginator module here?

Reply:
It seemed to work without it as well (this was just the output of `ng generate component`). Although most examples I saw did import the module so I added it in

Reply:
Yeah, we don't run the tests currently, but at runtime it would fail I think